Dapper/CustomPropertyTypeMap.cs

```diff
@@ -29,13 +29,13 @@ public CustomPropertyTypeMap(Type type, Func<Type, string, PropertyInfo> propert
         /// <param name="types">DataReader column types</param>
         /// <returns>Default constructor</returns>
         public ConstructorInfo FindConstructor(string[] names, Type[] types) =>

```

Comment:
FWIW we mismatch here on DefaultTypeMap vs. this for nullability - intentional?

Reply:
Almost certainly not